pkg/cli/environment.go

```diff
@@ -115,6 +117,7 @@ func (s *EnvSettings) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.RegistryConfig, "registry-config", s.RegistryConfig, "path to the registry config file")
 	fs.StringVar(&s.RepositoryConfig, "repository-config", s.RepositoryConfig, "path to the file containing repository names and URLs")
 	fs.StringVar(&s.RepositoryCache, "repository-cache", s.RepositoryCache, "path to the file containing cached repository indexes")
+	fs.UintVar(&s.MaxColWidth, "max-col-width", 80, "maximum column width for output table")
```

Comment:
This seems slightly confusing... `func (s *EnvSettings) AddFlags` adds these flags to the global flagset. It should be added where needed for now so it won't clobber the localized `--max-col-width` for `helm search repo` as well as `helm depedendency list`.

This also adds this flag to every single command, even if it doesn't produce table output.